src/Fulma/Common.fs

```diff
@@ -95,6 +95,51 @@ module Size =
         | IsMedium -> Classes.IsMedium
         | IsLarge -> Classes.IsLarge
 
+[<AutoOpen>]
+module Modifier =
+
+    module Classes =
+        module BackgroundColor =
+            let [<Literal>] Black = "has-background-black"
+
+        module TextColor =
+            let [<Literal>] Black = "has-text-black"
+
+    type IBackgroundColor =
```

Comment:
I think `IBackgroundColor` and `ITextColor` can be replace by using `IColor` directly.

They all use the same [`$colors` variable](https://github.com/jgthms/bulma/blob/master/sass/base/helpers.sass#L97-L106).

Also, by using the `IColor` users will only need to add once their custom color versus adding it 3 times.

Also, because both `IBackgroundColor` and `ITextColor` implement the `Black` case, you will need to write `IBackgroundColor.Black` where you can write `ITextColor.Black` or just `Black` for the later.

Reply:
By using `IColor` we will derivated a bit from Bulma semantic because to use any color user will need to wirte `Is[MyColor]` but I am fine with it.

Can be read has my "element background color is black" if we write something like `Modifiers [ Background IsBlack ]`

Reply:
I updated the code to what I think you mean.
Once I get the 'skeleton' right, I will go ahead and finish it.